api/ccapi/include/tensor_dim.h

```diff
@@ -54,6 +54,7 @@ class TensorDim {
   enum class DataType {
     QINT4,  /** quantized int 4*/
     QINT8,  /** quantized int 8*/
+    BCQ,    /** binary-code-based quantized*/
```

Comment:
trivial, but:
```C++
/** binary-code-based quantization*/
```
?

Reply:
I followed the upper naming conventions (quantized int 4, int 8).
If you want, we can change all of it :) 
```
QINT4,  /** int4 quantization*/
QINT8,  /** int8 quantization*/
BCQ,    /** binary-code-based quantization*/
```

Reply:
Ah, I see